pandas/core/indexes/accessors.py

```diff
@@ -157,6 +157,32 @@ class TimedeltaProperties(Properties):
     """
 
     def to_pytimedelta(self):
+        """
+        Return array of Timedeltas as `datetime.timedelta`.
+
+        Return an array of the Timedeltas in `datetime.timedelta` format with
+        the same length as the original Series.
+
+        Returns
+        -------
+        a : numpy.ndarray
+            1D array containing data with `datetime.timedelta` type.
+
+        Examples
+        --------
+        >>> s = pd.Series(pd.to_timedelta(np.arange(5), unit='d'))
+        >>> s
+        0   0 days
+        1   1 days
+        2   2 days
+        3   3 days
+        4   4 days
+        dtype: timedelta64[ns]
+        >>> s.dt.to_pytimedelta()
```

Comment:
can you add a blank line before this case

Reply:
Added.